editor/export/editor_export_platform.cpp

```diff
@@ -2466,6 +2468,35 @@ Array EditorExportPlatform::get_current_presets() const {
 	return ret;
 }
 
+Variant EditorExportPlatform::get_project_setting(const Ref<EditorExportPreset> &p_preset, const StringName &p_name) {
```

Comment:
This method would really useful to have available to GDExtension, for use in `EditorExportPlugin`s. More specifically, in the [godot_openxr_vendor](https://github.com/GodotVR/godot_openxr_vendors) extension, it would allow us to eliminate some export settings for things we can know from project settings, but it's only safe to check with the feature tags for that specific export preset.

So, I think this should be exposed, either right on this class, or as a part of `EditorExportPreset`, or even a convenience method on `EditorExportPlugin` - whatever would make the nicest API

Reply:
Exposed it as `EditorExportPreset::get_project_setting`.